tests/test_ec2/test_key_pairs.py

```diff
@@ -77,6 +79,7 @@ def test_key_pairs_create_dryrun_boto3():
 
 
 @mock_ec2
+@freeze_time("2014-01-01 05:00:00")
```

Comment:
Best to avoid freezing time unless absolutely an essential part of the test, as it doesn't work when we run the tests in server mode.